lib/gollum/templates/layout.mustache

```diff
@@ -15,11 +15,15 @@
   <![endif]-->
 
   <script>
-      var baseUrl = '{{base_url}}';
-      var uploadDest   = '{{upload_dest}}';
-      {{#page}}
-      var pageFullPath = '{{url_path}}';
-      {{/page}}
+	var baseUrl = '{{base_url}}';
+	var uploadDest = 'uploads';
+	var perPageUploads = '{{per_page_uploads}}';
```

Comment:
Wait, I'm not sure for the need for this now that the upload destination is being set using the referrer.

Reply:
In the new setup the frontend and the backend determine the upload path separately. The frontend only uses this for showing/inserting the upload path, but it is no longer leading for determining the actual path (which happens in the backend). I like this approach better than having to set an instance variable for each route that possibly supports uploading (e.g., create, edit, show).

Reply:
> The frontend only uses this for showing/inserting the upload path

Ah, great! I didn't realize it was for that.